src/scripts/git_hooks/install_git_hooks.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
On which system is this necessary?  On linux systems, /bin/sh is /bin/bash.

Reply:
Ubuntu for example, where sh is dash. bash is not a POSIX compliant shell.